src/app/components/profile/profile.component.spec.ts

```
@@ -0,0 +1,373 @@
+    component = fixture.componentInstance;
+    authService = TestBed.inject(AuthService) as jasmine.SpyObj<AuthService>;
+    snackBar = TestBed.inject(MatSnackBar) as jasmine.SpyObj<MatSnackBar>;
+  });
+
+  beforeEach(() => {
+    // Reset spies before each test
+    mockAuthService.getCurrentUser.calls.reset();
+    mockAuthService.getCurrentUserProfile.calls.reset();
+    mockAuthService.updateProfile.calls.reset();
+    mockSnackBar.open.calls.reset();
+  });
```

Biome (1.9.4)

[error] Disallow duplicate setup and teardown hooks.

Disallow beforeEach duplicacy inside the describe function.

(lint/suspicious/noDuplicateTestHooks)